cmd/account/account-assign.go

```diff
@@ -141,13 +143,15 @@ func (o *accountAssignOptions) run() error {
 
 	if o.payerAccount == defaultPayer {
 		destinationOu = defaultBYOCEnv
+		rootId = profileRootId2
 		_, err = awsClient.MoveAccount(inputMove)
 		if err != nil {
 			fmt.Println(err.Error())
 			return err
 		}
 	} else if o.payerAccount == nonDefaultPayer {
 		destinationOu = defaultNonBYOCEnv
+		rootId = profileRootId1
 		_, err = awsClient.MoveAccount(inputMove)
```

Comment:
No need to execute `MoveAccount()` twice. You can update `inputMove` and just run `MoveAccount()` outside of the if statement.